recipes/boost/all/conanfile.py

```diff
@@ -53,7 +53,7 @@ class BoostConan(ConanFile):
         "lzma": [True, False],
         "zstd": [True, False],
         "segmented_stacks": [True, False],
-        "debug_level": [i for i in range(1, 14)],
+        "debug_level": [i for i in range(0, 14)],
```

Comment:
I don't think this option changes the generated package.
=> remove it in `package_id(self)`?

Reply:
It is already removed there :)

Reply:
hehe, sorry.
I thought for a minute you were adding the option.